src/k8sconfiguration/azext_k8sconfiguration/custom.py

```diff
@@ -0,0 +1,176 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+
+from azext_k8sconfiguration.vendored_sdks.models.source_control_configuration_py3 import SourceControlConfiguration
+from azext_k8sconfiguration.vendored_sdks.models.helm_operator_properties import HelmOperatorProperties
+
+
+def show_k8sconfiguration(client, resource_group_name, cluster_name, name, cluster_type='connectedClusters',
+                          configuration_type='sourceControlConfiguration', api_version='2019-11-01-preview'):
+    # Determine ClusterRP
+    if cluster_type.lower() == 'connectedclusters':
+        cluster_rp = 'Microsoft.Kubernetes'
+    elif cluster_type.lower() == 'managedclusters':
+        cluster_rp = 'Microsoft.ContainerService'
+    else:
+        raise CLIError("Invalid cluster-type.  Supported values are 'connectedClusters' and 'managedClusters'.")
+
+    # Validate ConfigurationType
+    if configuration_type.lower() != 'sourcecontrolconfiguration':
```

Comment:
Do we need to support more values for `configuration_type` in the future ?

Reply:
Yes, we intend to support more configuration types in the future